ezbake/config/conf.d/auth.conf

```diff
@@ -1,39 +1,116 @@
 authorization: {
-    # TODO: Fill these in with real default rule definitions - SERVER-846.
-    # Putting just the ones for the certificate authority in for now since
-    # these are always evaluated by trapperkeeper-authorization and are
-    # needed to complete a successful agent run.
+    version: 1
     rules: [
         {
+            # Allow nodes to retrieve their own catalog
+            match-request: {
+                path: "^/puppet/v3/catalog/([^/]+)$"
+                type: regex
+                method: [get, post]
+            }
+            allow: "$1"
+            sort-order: 500
+            name: "puppetlabs catalog"
```

Comment:
The order of the rules in this file is different that the old order, since we now have the notion of sorting the rules and a warning will be printed out if they're not sorted (on sort-order & name) in the file.

I tried to choose meaningful names for the rules, and then sorted them alphabetically (since they all have the same sort-order 500).